### ISSUE TITLE
Links in search results are not visible on hover

### ISSUE THREAD

**Issue by salim-b** [gethyas/doks-core#66]
## Description

When content contains links (rendered to `<a>`) ***and*** the [`<!--more-->` summary divider](https://gohugo.io/content-management/summaries/#manual-summary-splitting) is used to fine-tune the content's `.Summary`, those links won't be visible in search result ***on hover***.

Screencast:

[Bildschirmaufzeichnung vom 24.10.2023, 01:34:41.webm](https://github.com/gethyas/doks-core/assets/20040931/ddd351c1-b335-4072-b516-1bb81dd8a1b3)

This is because some Bootstrap

```css
a {
  color: rgba(var(--bs-link-color-rgb), var(--bs-link-opacity, 1));
}
```

rule has precedence over Doks'

```css
.search-result:hover .card,
.search-result.selected .card {
  background-color: #5d2f86;
  color: #fff; }
```

rule, I think.

But since you're more familiar with Doks styling (and way more proficient in CSS) than me, I let this up to you, @h-enk :slightly_smiling_face: .

### Steps to reproduce

1. Add the following content to a Markdown `content` file:

   ```md   
   Post content [with a link to Doks](https://getdoks.org).
   
   <!--more-->
   ```

2. Search for the above content.

3. Hover over the search result.

### Expected result

Links should also get color-inverted in the same way as the rest of the search result card.

### Actual result

Links do not get color-inverted on hover.

### Environment

There is no `check` script in [Doks default `package.json`](https://github.com/gethyas/create-hyas/blob/5d93dce3cc21db6c530d085d964ae8ff661f9bd6/template-doks/package.json#L7-L26)...

**Comment by h-enk** [comment]
> There is no check script in [Doks default package.json](https://github.com/gethyas/create-hyas/blob/5d93dce3cc21db6c530d085d964ae8ff661f9bd6/template-doks/package.json#L7-L26)...

Yes thx, it should be `info` — fixed: https://github.com/gethyas/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.md

**Comment by h-enk** [comment]
Fixed w/ 328559b81ec972d680681d7bdb866d2a61dd8f5c

![Snag_88bcf1](https://github.com/gethyas/doks-core/assets/3902872/69b75e16-f0ce-4562-813a-c05c59a13522)

Also, made both the "Example Post" (stretched, global) link (the card) and the "with a link to Doks" (specific link) clickable
